### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by UsirusenIkon**
## Requirements👌
- We kept the general layout of the wireframe, this is the only mandatory requirement.
- Now you should improve the look and feel of the application, adding the styles of your choice.
- We used plain CSS.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,20 +7,21 @@
...
+          <h2>Refresh Leaderboard</h2>
         </ul>
       </div>
     </div>
     <form class="form flex" >
-      <span>Add Your Score</span>
+      <span>Add Your <span class="color">Score</span></span>
       <label for="Name">
         <input class="leader user" name="Name" type="text" placeholder="your name" maxlength="30" required>
       </label>
```
Comment (GzimAsani) [inline src/index.html:27]: - [ ] Kindly align `Recent scores` and `Add new scores` so they look similar to the given wireframe

Yours:
image

Wireframe:
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @UsirusenIkon          :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your styling, I really like it+1: 
 
 ## Required Changes 

- [x]  On your readme file, please let the user know how can they run your project, example `npm start`
- [x] Check the comments below :arrow_double_down: 




 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (UsirusenIkon) [reply src/index.html:27]: Hey @GzimAsani, Thank you for the review I'll do well to put the changes for the next reviewer.👌👍

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @UsirusenIkon 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**